wled00/data/index.js

```diff
@@ -627,10 +630,12 @@ function populateSegments(s)
 		noNewSegs = false;
 	}
 	for (var i = 0; i <= lSeg; i++) {
-	updateLen(i);
-	updateTrail(d.getElementById(`seg${i}bri`));
-	if (segCount < 2) d.getElementById(`segd${lSeg}`).style.display = "none";
+		updateLen(i);
+		updateTrail(d.getElementById(`seg${i}bri`));
+		d.getElementById(`segr${i}`).style.display = "none";
```

Comment:
This results in `Uncaught TypeError: Cannot read properties of null (reading 'style')` if you delete any but the last segment (e.g. you have multiple segments and delete seg. 0). updateLen() and updateTrail() just return on null pointer.

Reply:
Will look into it. It didn't occur to me to delete segment 0.